### ISSUE TITLE
Is hierarchical softmax used in Relevance-Based-Word-Embeddings?

### ISSUE THREAD

**Issue by rossbrown9879** [teinhonglo/Information-retrieval#15]
I suppose that in maximum likelihood estimation based model for training relevance based word embedding. In the paper (https://dl.acm.org/doi/10.1145/3077136.3080831), the author mentions that we can not use negative sampling in this training. We need to approximate the softmax here. Have you used hierarchical softmax in your implementation anywhere?
